src/snyk/snykOss/services/vulnerabilityCount/vulnerabilityCountProvider.ts

```diff
@@ -20,15 +29,39 @@ export class ModuleVulnerabilityCountProvider {
     };
 
     if ([Language.TypeScript, Language.JavaScript, Language.PJSON].includes(language)) {
+      // TODO use LS when OSS is moved to LS
       const ossResult = this.ossService.getResultArray();
       if (!ossResult) {
         return notCalculated;
       }
 
       return this.mapOssResult(module, ossResult);
-    } else if (language == Language.HTML) {
-      const testedModuleInfo = await this.npmModuleInfoFetchService.getModuleVulnerabilityInfo(module);
-      return this.mapTestedImportedModule(testedModuleInfo);
+    } else if (language == Language.HTML && module.loc) {
+      const uri = this.uriAdapter.file(fileName).toString();
+      const doc: LSPTextDocument = this.textDocumentAdapter.create(uri, 'HTML', 1, '');
+      const line = module.loc.start.line - 1;
+      const param = {
+        textDocument: { uri: doc.uri },
+        range: {
+          start: { line: line, character: module.loc.start.column },
+          end: { line: line, character: module.loc.end.column },
+        },
+      };
+      const inlineValues: InlineValueText[] = await this.lca
+        .getLanguageClient()
+        .sendRequest('textDocument/inlineValue', param);
+
+      if (inlineValues.length > 0) {
+        return {
+          name: module.name,
+          version: module.version,
+          fileName: module.fileName,
+          line: module.line,
+          range: module.loc,
+          count: inlineValues[0].text,
+          hasCount: true,
+        } as unknown as ModuleVulnerabilityCount;
```

Comment:
This is a bit of a cheat, we should probably return an object of the expected type. Is it not possible?

Reply:
I think it is :)

Reply:
I'll employ your help for this, Jason :D

Reply:
Done